CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.2.0 (2020-07-21)
+
+- Added ability to hide restricted items in the sidebar navigation. [#79](https://github.com/blackbaud/skyux-docs-tools/pull/79)
```

Comment:
Can you add "the" before ability? Otherwise looks good!

Reply:
done!